python/lsst/ts/standardscripts/auxtel/daytime_checkout/atpneumatics_checkout.py

```diff
@@ -124,22 +138,59 @@ async def run(self):
 
         await self.atcs.enable_ataos_corrections()
 
-        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(timeout=5)
-        self.log.info(
-            f"M1 Air pressure with enabled ATAOS corrections is {m1_pressure.pressure:0.0f} Pascals"
+        # Sleep to allow M1 to arrive at commanded pressure
+        await asyncio.sleep(self.ataos_enabled_delay)
+
+        m1_commanded_pressure = (
+            await self.atcs.rem.ataos.evt_m1CorrectionCompleted.aget(
+                timeout=STD_TIMEOUT
+            )
         )
 
+        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(
+            timeout=STD_TIMEOUT
+        )
+
+        # Check that M1 pressure arrived to ATAOS commanded pressure within
+        # tolerance
+        if (
+            m1_commanded_pressure.pressure * (1.0 - self.ataos_pressure_tolerance)
```

Comment:
you may want to specify that `self.ataos_pressure_tolerance` is a "relative tolerance" I initially thought this was absolute value and got confused. But I agree it makes sense to use relative. So maybe just rename this `self.pressure_relative_tolerance` or `self.pressure_tolerance_relative`?

Reply:
Change accepted.